404.html

```diff
@@ -3,12 +3,20 @@
 title: "Error 404"
 ---
 <div class="gray-wrapper">
-    <div class="container spaced">
-        <div class="row">
-            <div class="col">
-                <h2>Page not found :(</h2>
-                <p>The requested page could not be found.</p>
-            </div>
-        </div>
-    </div>
-</div>
+	<div class="container spaced">
+		<div class="row">
+			<div class="col">
+				<h2>Page not found :(</h2>
+				<p>The requested page could not be found.</p>
+			</div>
+			<div class="col-xs-12 col-sm-6 col-md-6 vcenter">
+				<div class="embed-responsive embed-responsive-16by9 csh-404-video">
+					<iframe class="embed-responsive-item" src="https://www.youtube.com/embed/t3otBjVZzT0?autoplay=1" frameborder="0" allowfullscreen></iframe>
```

Comment:
These `embed-responsive` and `csh-404-video` classes don't seem to be associated with or do anything. Are these necessary?

Reply:
Fixed, thanks!